src/project.ts

```diff
@@ -21,13 +21,84 @@ export interface Project {
   rootPackage: Package;
   workspacePackages: Record<string, Package>;
   isMonorepo: boolean;
+  releaseInfo: ReleaseInfo;
+}
+
+/**
+ * Information about the release of the root package of a monorepo extracted
+ * from its version string.
+ *
+ * @property releaseDate - The release date.
+ * @property releaseNumber - The release number (starting from 1).
```

Comment:
Any reason to start at one rather than zero? 

I guess it is conventional to start at zero with the "RC__" syntax. But I've more typically seen it start at zero when the build number is just a part of the semver version. But we have both representations in different places.

Reply:
My thought is that we could use the release number to generate the release notes, so we say e.g. "August 26, 2022 (R1)" for the very first release instead of "August 26, 2022 (R0)". For that reason 1 seems like a more natural starting number than 0, but we can technically do whatever we want here.

Reply:
This is now outdated following discussion+changes, resolving.